### PR TITLE
T018 clean footer

### PR DESCRIPTION
**Pull request by tiaruzica**

### CODE REVIEW

**Review by johnchen383 — APPROVED** [review]
LGTM. Feel free to merge to main :))

File: src/components/Footer.tsx
Comment (KennethZhu904) [inline src/components/Footer.tsx:11]: Bit a too much space, pls remove them.

File: src/screens/Landing.tsx
Comment (KennethZhu904) [inline src/screens/Landing.tsx:38]: Just remove one space and that should be fine

File: src/widgets/Image.tsx
Comment (KennethZhu904) [inline src/widgets/Image.tsx:18]: Remove one space and it should be fine.

File: src/widgets/YoutubeVideo.tsx
Comment (KennethZhu904) [inline src/widgets/YoutubeVideo.tsx:13]: please get rid of that comment - if you don't need it anymore.

**Review by KennethZhu904 — APPROVED** [review]
LGTM - once you fix some of my changes, you can merge them :)

**Pull request merged**
